src/lvl_script.c

```diff
@@ -349,6 +351,16 @@ const struct NamedCommand script_operator_desc[] = {
 /******************************************************************************/
 DLLIMPORT long _DK_script_support_send_tunneller_to_appropriate_dungeon(struct Thing *creatng);
 /******************************************************************************/
+static int filter_criteria_type(long desc_type)
+{
+    return desc_type & 0x0F;
```

Comment:
What does this mean?

Reply:
unpacking kind of criteria from packet format